data/pilots/galactic-empire/tie-sa-bomber.json

```diff
@@ -122,8 +121,7 @@
       "caption": "Scimitar Leader",
       "initiative": 4,
       "limited": 1,
-      "cost": 5,
-      "loadout": 12,
+      "cost": 3,
```

Comment:
This is the standard Rhymer, so I think he should lose his slots? Although I guess the Bombers are probably just all bugged right now while we wait for the fixer-upper drop.

Reply:
Yeah that was my thoughts, just wait for the obvious incoming fixes